### DISCUSSION TITLE
Question: When is the next release coming?

### DISCUSSION THREAD

**Discussion by mabod** [geany/geany#3404] [Q&A] [answered]
the last release 1.38 is from October 2021.
When will the next release come? There have been plenty of commits in the meantime and the list of pull requests is long.

**Comment by elextr** [comment]
@b4n ???

**Comment by mabod** [comment]
What is this? I only see a repo with a geany fork with the latest commit in 2016.

**Comment by elextr** [comment]
Huh? I was asking @b4n if there was any plan for a release.

**Comment by mabod** [comment]
Oh man. It was to early in my morning. ;-)

**Comment by b4n** [comment]
@elextr well, if it's in a shape we all like, sure.  AFAIKT the [milestone](/geany/geany/milestone/20) is not cleared, so maybe we should look there. But surely there's something to release indeed :slightly_smiling_face:

**Comment by b4n** [comment, marked as answer]
So, definite answer is "last Thursday" :wink:
